tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+class BottomBar(Base):
+    _remove_locator = (By.ID, 'remove')
+    _done_locator = (By.ID, 'done')
+
```

Comment:
One way to debug that you're waiting on the right values:
```py
for i in range(0, 20)
    print i, view.location['y'], bottom_bar_position["y"]
```

THen you run the tests and see how these values evolves